packages/paste-website/src/pages/components/badge/index.mdx

```diff
@@ -91,55 +99,98 @@ export const pageQuery = graphql`
 
 <content>
 
+<LivePreview scope={{Box, Badge, InformationIcon, ErrorIcon, WarningIcon, SuccessIcon, NewIcon}} language="jsx">
+  {`<Box display="flex" columnGap="space40" rowGap="space60" flexWrap="wrap">
+  <Badge as="span" variant="neutral">
+    Neutral
+  </Badge>
+  <Badge as="span" variant="warning">
+    Warning
+  </Badge>
+  <Badge as="span" variant="error">
+    Error
+  </Badge>
+  <Badge as="span" variant="success">
+    Success
+  </Badge>
+  <Badge as="span" variant="new">
+    New
+  </Badge>
+  <Badge as="span" variant="decorative10">
+    Decorative 10
+  </Badge>
+  <Badge as="span" variant="decorative20">
+    Decorative 20
+  </Badge>
+  <Badge as="span" variant="decorative30">
+    Decorative 30
+  </Badge>
+  <Badge as="span" variant="decorative40">
+    Decorative 40
+  </Badge>
+  <Badge as="span" variant="neutral_counter">
+    1
+  </Badge>
+  <Badge as="span" variant="error_counter">
+    1
+  </Badge>
+</Box>
+`.trim()}
+</LivePreview>
+
 ## Guidelines
 
-### About Badge
+## About Badge
+
+A Badge can be used to label a piece of UI for quick identification.
+It can be used in a wide variety of situations, such as labeling Beta product features
+or an attribute such as “Inbound” or “Outbound”. A Badge can also be used to show a count,
+such as the number of new messages.
+
+A Badge should contain text content and can optionally contain an Icon.
+Text length should be kept short.
 
-A Badge can be used to label a piece of UI for quick identification and navigation. It can be used in a wide variety of situations such as labelling Beta product features or a state, for example a status.
+### Accessibility
 
-A Badge should contain text content and can optionally contain smaller graphical elements such as Icons and Avatars. Text length should be kept short.
+- Ensure that Badge text is clear enough to understand immediately without having to rely on color to communicate meaning.
+  For example, an error badge should say “Token missing” rather than just “Token”, so the error is communicated via the Badge label.
+
```

Comment:
I think this extra line between the bullets is putting too much space between them.

Reply:
u right u right